src/silvistat/storage.py

```diff
@@ -1,68 +1,96 @@
 import pdal
 import tiledb
 import numpy as np
+from math import floor
 
 from . import Bounds
 
 class Storage(object):
-
-    def __init__(self, dirname, filename, config=None, cell_size=30,
+    """ Handles storage of shattered data in a TileDB structure. """
+    def __init__(self, dirname, filename, config= None, cell_size=30,
                  chunk_size=16):
         self.dirname = dirname
         self.src_file = filename
         self.config = config
         self.cell_size = cell_size
         self.chunk_size = chunk_size
+        ## TODO insert config here
+        self.ctx = tiledb.default_ctx()
 
+        self.atts = None
         self.schema = None
         self.tdb = None
         self.bounds = None
 
     def __inspect_file(self):
-        r = pdal.Reader(self.filename)
+        """
```

Comment:
This PDAL stuff doesn't seem like it should be long in this class. Should it be a freestanding utility method or something?